src/cdk/stepper/step-header.ts

```diff
@@ -11,9 +11,6 @@ import {FocusableOption} from '@angular/cdk/a11y';
 
 @Directive({
   selector: '[cdkStepHeader]',
-  host: {
```

Comment:
I think we'll have to leave this, because the CDK is used for more than Material.